plugins/woocommerce/includes/cli/class-wc-cli-com-command.php

```diff
@@ -16,6 +16,8 @@
  * @package WooCommerce
  */
 class WC_CLI_COM_Command {
+	const APPLICATION_PASSWORD_SECTION_URL = 'https://woocommerce.com/.com/my-account/#application-passwords';
```

Comment:
This URL gives me a 404. Is it incorrect or just hasn't been enabled yet?

Reply:
fixed in 352cc7e7d51916a61e481d6682dc46f3d00af4a5